sass/layout/_sidebar-left.scss

```diff
@@ -860,6 +966,18 @@
         height: 18px;
     }
 
+    .SidebarChannel div.SidebarChannelLinkLabel_wrapper {
```

Comment:
Why use this nesting? Can't we just assume that any `.SidebarChannelLinkLabel_wrapper` is part of a `.SidebarChannel`?

Reply:
Sure, I can remove this.